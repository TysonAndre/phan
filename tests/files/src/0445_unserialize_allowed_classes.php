<?php

$serialized = serialize(new stdClass());
unserialize($serialized, []);
unserialize($serialized, ['allowed_classes' => [['stdClass']]]);
unserialize($serialized, ['allowed_classes' => ['stdClass']]);
