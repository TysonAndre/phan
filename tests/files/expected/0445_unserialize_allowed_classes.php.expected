%s:5 PhanTypeMismatchArgumentInternal Argument 2 ($options) is ['allowed_classes'=>[['stdClass']]] of type array{allowed_classes:array{0:array{0:'stdClass'}}} but \unserialize() takes array{allowed_classes?:bool|string[]}
